content/docs/user-guide/components/reference/physx/prismatic-joint.md

```diff
@@ -0,0 +1,65 @@
+---
+linkTitle: PhysX Prismatic Joint
+description: ' Lean to use the Open 3D Engine PhysX Prismatic Joint component. '
+title: PhysX Prismatic Joint Component
+---
+
+
+
+With the **PhysX Prismatic Joint** component, you can create a dynamic prismatic joint that constrains an entity to the joint keeping the same rotation but allowing to move freely along one axis.
+
+## PhysX Prismatic Joint component properties 
+
+![Properties of the PhysX Prismatic Joint component](/images/user-guide/physx/physx/ui-physx-prismatic-joint-component.png)
+
+**Local Position**
+Specify the position of the joint relative to the entity transform.
+
+**Local Rotation**
+Specify the rotation of the joint relative to the entity transform.
+
+**Lead Entity**
+Specify the parent entity that will drive the joint.
+
+**Breakable**
+When enabled, the joint will break if sufficient force is applied. Enabling **Breakable** exposes the **Maximum Force** and **Maximum Torque** properties.
+**PhysX Dynamic Rigid Body** components that have their **Compute Mass** property enabled might have very large mass values. If the entity containing the joint component or the leader entity have their **Compute Mass** property enabled, the **Maximum Force** and **Maximum Torque** properties might require very high values to resist breaking.
```

Comment:
suggestion: Is it worth having an example of the very high value? Something like `e.g. 100000N`?

Reply:
I just meant have an actual number (what does 'very high value' mean in this context?)

Reply:
I think the sentence just refers to the relationship between the `mass` value and the `maximum force/torque`, the higher the `mass` is, the higher the `maximum force/torque` needs to be.